Assets/LeapMotionModules/GraphicRenderer/Scripts/LeapGraphicGroup.cs

```diff
@@ -51,6 +53,16 @@ public partial class LeapGraphicGroup : ISerializationCallbackReceiver {
 
     #region PUBLIC RUNTIME API
 
+    public string name {
+      get {
+        return _groupName;
+      }
+    }
+
+    /// <summary>
+    /// Gets the renderer this group is attached to.  The set method is 
+    /// for internal use only and should not be used.
```

Comment:
not important for 0.1.0 but doing that 'internal interface' thing is a good way to prevent this sort of commentary from being necessary